test/e2e/azure_logcollector.go

```diff
@@ -156,6 +156,9 @@ func collectLogsFromNode(ctx context.Context, managementClusterClient client.Cli
 		errors = append(errors, kinderrors.AggregateConcurrent(windowsInfo(execToPathFn)))
 		errors = append(errors, kinderrors.AggregateConcurrent(windowsK8sLogs(execToPathFn)))
 		errors = append(errors, kinderrors.AggregateConcurrent(windowsNetworkLogs(execToPathFn)))
+		errors = append(errors, kinderrors.AggregateConcurrent(windowsCrashDumpLogs(execToPathFn)))
+		errors = append(errors, sftpCopyFile(controlPlaneEndpoint, hostname, sshPort, "/c:/crashdumps.tar", filepath.Join(outputPath, "crashdumps.tar")))
```

Comment:
does this need to be `kinderrors.AggregateConcurrent`  or since its just one func the error is caught properly

Reply:
is forward slash in `/c:/crashdumps.tar` needed?

Reply:
In my testing the errors where caught properly.
Yes the forward slash is needed (took me a bit to figure that out :P)

Reply:
When you enter an sftp session on Windows the local directory is set to the user directory (c:\Users\{user}) and the forward slash specifies an absolute path.

Reply:
interesting!